website/src/pages/terms-of-service.tsx

```diff
@@ -1,148 +1,27 @@
-import { Container, Heading } from "@chakra-ui/react";
+import { Box, Heading, Stack } from "@chakra-ui/react";
```

Comment:
Could you pull out the terms of service changes since that's in a separate PR?

Reply:
Of course, sorry I thought I took them out.